vars.tf

```diff
@@ -43,3 +43,11 @@ variable "metrics_password_key" {
   default     = "password"
   description = "Key in metrics secret containing password for Prometheus"
 }
+
+variable "external_labels" {
+  type = map(string)
+  default = {
+    cluster = "cloud"
```

Comment:
Is this really what we want as default value?

Reply:
That was the "default" value for cluster previously. I agree that it's not a good default value. 😜 Just noticed a couple of more things that needs to be updated though so this will be replaced anyway.